app/controllers/application_controller.rb

```diff
@@ -50,6 +50,8 @@ def check_privacy_policy_accepted
     policy = PrivacyPolicy.current
     return if policy.nil?
 
+    return if request.format == "application/json"
```

Comment:
this is technically unrelated to this PR, but it feels like it's probably a good idea